include/xrpl/basics/TaggedCache.h

```diff
@@ -210,7 +210,7 @@ class TaggedCache
         std::vector<SweptPointersVector> allStuffToSweep(m_cache.partitions());
 
         clock_type::time_point const now(m_clock.now());
-        clock_type::time_point when_expire;
+        clock_type::time_point expire_before_this;
```

Comment:
Did you _only_ rename this without changing how it's used? Could you add a comment here explaining how it's used / what it means?